homeassistant/components/directv/manifest.json

```diff
@@ -4,5 +4,12 @@
   "documentation": "https://www.home-assistant.io/integrations/directv",
   "requirements": ["directpy==0.6"],
   "dependencies": [],
-  "codeowners": []
+  "codeowners": [],
```

Comment:
Can you add yourself as a code owner as you're making significant changes to this integration.